cmd/tailscale/cli/debug.go

```diff
@@ -190,6 +192,16 @@ var debugCmd = &ffcli.Command{
 			Exec:      runDebugDERP,
 			ShortHelp: "test a DERP configuration",
 		},
+		{
+			Name:      "capture",
+			Exec:      runCapture,
+			ShortHelp: "streams pcaps for debugging",
+			FlagSet: (func() *flag.FlagSet {
+				fs := newFlagSet("capture")
+				fs.StringVar(&captureArgs.outFile, "o", "<wireshark>", "where to stream the pcap, or <wireshark> to start wireshark")
```

Comment:
that's a kinda hard sentinel value to document due to the different quoting rules between operating systems & shells. and it'll render in flag help doc as if it's a parameter. 

also, what happened to "-" for stdout?

Reply:
"-" for stdout works! I guess we should fill in `LongHelp` with all the details

Reply:
As for triggering the wireshark flow, I'll move that to a positional parameter

Reply:
I mean you didn't document "-" for stdout.